starwizard-services/src/main/java/io/stardog/starwizard/services/stripe/StripeService.java

```diff
@@ -67,6 +67,25 @@ public StripeService(AsyncService asyncService,
         }
     }
 
+    /**
+     * Creates a new SetupIntent object.
+     * @params params   Parameters to pass along to Stripe
+     * @return created  Stripe Setup Intent object
+     */
+    public SetupIntent createSetupIntent(@Nullable Map<String, Object> params) {
+
+        try{
+            if(params != null){
```

Comment:
I think you can make this a little bit prettier with:

```
params = params != null ? params : new HashMap<>();
return SetupIntent.create(params);
```

That way we don't see two multiple paths to invoke the Stripe call. Not really a big deal though.